packages/teleport/src/components/Toggle/Toggle.tsx

```diff
@@ -63,7 +77,7 @@ const StyledSlider = styled.div`
 const StyledInput = styled.input.attrs({ type: 'checkbox' })`
```

Comment:
Let's add this at the end of styles for `StyledInput` to visually differentiate disabled Toggles from enabled ones:

```typescript
  &:disabled + ${StyledSlider} {
    background: ${props => props.theme.colors.primary.light};

    &:before {
      background: ${props => props.theme.colors.grey[700]};
    }
  }
```

Reply:
Good idea, thanks!